### PR TITLE
LPS-139992 User cannot assign Job Title when creating Account Users

### PR DESCRIPTION
**Pull request by erickmonteiroo**
@drewbrokke @pei-jung see [this](https://github.com/brianchandotcom/liferay-portal/pull/109777#issuecomment-968509077) comment.

### CODE REVIEW

**Comment by erickmonteiroo** [comment]
ci:test:account

File: modules/apps/account/account-service/src/main/java/com/liferay/account/service/impl/AccountEntryUserRelServiceImpl.java [hunk 1]
```diff
@@ -84,6 +108,28 @@ public void addAccountEntryUserRels(
...
+		_modelResourcePermission.check(
+			getPermissionChecker(), accountEntryId, ActionKeys.MANAGE_USERS);
+
+		AccountEntryUserRel accountEntryUserRel =
+			accountEntryUserRelLocalService.addPersonTypeAccountEntryUserRel(
+				accountEntryId, creatorUserId, screenName, emailAddress, locale,
+				firstName, middleName, lastName, prefixId, suffixId, jobTitle);
+
+		_userLocalService.updateJobTitle(
+			accountEntryUserRel.getAccountUserId(), jobTitle);
```
Comment (drewbrokke) [inline modules/apps/account/account-service/src/main/java/com/liferay/account/service/impl/AccountEntryUserRelServiceImpl.java:128]: This should be removed, right? Same with line 78

File: modules/apps/account/account-service/src/main/java/com/liferay/account/service/impl/AccountEntryUserRelServiceImpl.java [hunk 2]
```diff
@@ -54,7 +56,29 @@ public AccountEntryUserRel addAccountEntryUserRel(
...
+		_modelResourcePermission.check(
+			getPermissionChecker(), accountEntryId, ActionKeys.MANAGE_USERS);
+
+		AccountEntryUserRel accountEntryUserRel =
+			accountEntryUserRelLocalService.addAccountEntryUserRel(
+				accountEntryId, creatorUserId, screenName, emailAddress, locale,
+				firstName, middleName, lastName, prefixId, suffixId, jobTitle);
+
+		_userLocalService.updateJobTitle(
+			accountEntryUserRel.getAccountUserId(), jobTitle);
```
Comment (drewbrokke) [inline modules/apps/account/account-service/src/main/java/com/liferay/account/service/impl/AccountEntryUserRelServiceImpl.java:79]: This should be removed too

**Review by drewbrokke — CHANGES_REQUESTED** [review]
@erickmonteiroo we should now remove the jobtitle logic from the remote service.

**Pull request closed without merging**